CallStack (from HasCallStack):
  throwError, called at src/Malgo/Infer/Kind.hs:36:12 in malgo-2.0.0-inplace:Malgo.Infer.Kind
test/Malgo/InferSpec/errors/InvalidTypeSynonym.mlg: line 5, column 1 - line 8, column 1:
Invalid type application: Maybe Int32 cannot be applied to Int32